semcore/base-trigger/src/FilterTrigger.jsx

```diff
@@ -101,51 +127,105 @@ class RootFilterTrigger extends Component {
         use:aria-labelledby={undefined}
       >
         <NeighborLocation>
-          <SFilterTrigger
-            tag='button'
-            type='button'
-            w='100%'
-            size={size}
-            placeholder={placeholder}
-            empty={empty}
-            selected={!empty}
-            active={active}
-            disabled={disabled}
-            ref={triggerRef}
-            animationsDisabled
-            {...controlProps}
-          >
-            {addonTextChildren(
-              Children,
-              FilterTrigger.Text,
-              [FilterTrigger.Addon, FilterTrigger.Counter],
-              empty,
-            )}
-            {empty && <FilterTrigger.Addon tag={ChevronDown} />}
-          </SFilterTrigger>
-          {!empty && (
-            <SFilterTrigger
-              tag='button'
-              type='button'
-              size={size}
-              empty={empty}
-              selected
-              onClick={callAllEventHandlers(onClear, this.handleClear, this.handleStopPropagation)}
-              onKeyDown={this.handleStopPropagation}
-              disabled={disabled}
-              id={`igc-${uid}-filter-trigger-clear`}
-              aria-labelledby={`igc-${uid}-filter-trigger-clear ${id}`}
-              aria-label={getI18nText('clear')}
-            >
-              <FilterTrigger.Addon tag={Close} />
-            </SFilterTrigger>
+          {TriggerButton ?? (
```

Comment:
1. Do I understand correctly that I can't render any children without TriggerButton?
2. Is it possible to wrap TriggerButton? For example, add tooltip over TriggerButton?

Reply:
1. Yes, but this is current behaviour too. All your children will be rendered inside TriggerButton. But you shouldn't wrap them into TriggerButton, if you don't need to use custom clear button.
2. No. You could wrap all FilterTrigger or Content inside TriggerButton. For example,
``